ReText/dialogs.py

```diff
@@ -79,3 +79,28 @@ def handleTextChanged(self, value):
 			button.setEnabled(False)
 		else:
 			button.setEnabled(True)
+
+
+class urlPopup(QLabel):
```

Comment:
Please move this class to `webenginepreview.py`, since it's not a dialog, but part of the previewer. 

Also please capitalize the first letter: `UrlPopup`.